requirements.txt

```diff
@@ -1,4 +1,4 @@
-numpy==1.8.0
-numpydoc==0.4
-networkx==1.8.1
+numpy>=1.8.0
+numpydoc>=0.4
+networkx>=1.8.1
```

Comment:
+1 for only keeping the requirements in the setup.py.

See INSTALL_REQUIRES and EXTRAS_REQUIRE. While at it, this might be the time to change PyQt4 requirement in setup.py to AnyQt